CODE_OF_CONDUCT.md

```diff
@@ -1,46 +1,119 @@
-# Contributor Covenant Code of Conduct
+# Our Pledge
+In the interest of fostering an open, collaborative, and welcoming environment, we as contributors and maintainers of nf-core, pledge to making participation in our projects and community a harassment-free experience for everyone, regardless of:
 
-## Our Pledge
+- Age
+- Body size
+- Familial status
+- Gender identity and expression
+- Geographical location
+- Level of experience
+- Nationality and national origins
+- Native language 
+- Physical and neurological ability
+- Race or ethnicity
+- Religion
+- Sexual identity and orientation
+- Socioeconomic status
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+Please note that the list above is alphabetised and is therefore not ranked in any order of preference or importance. 
 
-## Our Standards
+# Code of Conduct at nf-core (v1.0) 
+#### Date: March 12, 2021
```

Comment:
I think that we can ditch this if we have a changelog (see comment at the bottom of this document).

```suggestion
```